ArchiSteamFarm/ASF.cs

```diff
@@ -197,8 +180,8 @@ internal static class ASF {
 					return null;
 				}
 
-				if (!string.IsNullOrEmpty(releaseResponse.ReleaseNotesInMarkdown)) {
-					string plainText = Utilities.MarkdownToText(releaseResponse.ReleaseNotesInMarkdown);
+				if (!string.IsNullOrEmpty(releaseResponse.MarkdownBody)) {
```

Comment:
Replace this with `ChangelogPlainText` that should work similar to `ChangelogHTML`, also cache `MarkdownDocument` with changelog only so call to `ChangelogPlainText` after `ChangelogHTML` would only render `MarkdownDocument` as `string` and skip expensive MD evaluation.